### ISSUE TITLE
cannot import name 'cost_or_value' from 'fava.template_filters' 

### ISSUE THREAD

**Issue by uherbst** [redstreet/fava_investor#89]
- I have a fresh python3.12.1 installation on a Mac (M2-Chip) from python.org
- pip3 install beancount fava fava-investor

Starting fava on my beancount-file, I got an error.
Same if I do "import fava_investor" in a python-commandline.

```
uherbst@umbp2  ~ $ python3
Python 3.12.1 (v3.12.1:2305ca5144, Dec  7 2023, 17:23:38) [Clang 13.0.0 (clang-1300.0.29.30)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import fava_investor
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/uherbst/Library/Python/3.12/lib/python/site-packages/fava_investor/__init__.py", line 12, in <module>
    from .common.favainvestorapi import FavaInvestorAPI
  File "/Users/uherbst/Library/Python/3.12/lib/python/site-packages/fava_investor/common/favainvestorapi.py", line 2, in <module>
    from fava.template_filters import cost_or_value
ImportError: cannot import name 'cost_or_value' from 'fava.template_filters' (/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/fava/template_filters.py)
>>>

```

Any ideas ?

Thank you
Uli

**Comment by uherbst** [comment]
Versions installed:
beancount                2.3.6
fava                     1.27.1
fava-investor            0.5.0

**Comment by redstreet** [comment]
Yes, this got fixed recently, but is not yet released. I'll make a release soon; install from top of the tree until then:

```
pip3 install git+https://github.com/redstreet/fava_investor
```

**Comment by redstreet** [comment]
Released 0.6.0.
